pkg/storage/test/authz_models.go

```diff
@@ -99,7 +99,7 @@ func ReadAuthorizationModelsTest(t *testing.T, datastore storage.OpenFGADatastor
 	}
 
 	opts = storage.ReadAuthorizationModelsOptions{
-		Pagination: storage.NewPaginationOptions(2, string(continuationToken)),
```

Comment:
Suggestion: rename: `continuationToken -> ulidOfModel` (or whatever you deem fit)